t/admin/consumers.t

```diff
@@ -322,3 +322,67 @@ GET /t
 {"error_msg":"not supported `POST` method for consumer"}
 --- no_error_log
 [error]
+
+
+
+=== TEST 10: add consumer with create_time and update_time(pony)
+--- config
+    location /t {
+        content_by_lua_block {
+            local t = require("lib.test_admin").test
+            local code, body = t('/apisix/admin/consumers',
+                ngx.HTTP_PUT,
+                [[{
+                     "username":"pony",
+                     "desc": "new consumer",
+                     "create_time": 1602883670,
```

Comment:
So this means the manager api will append create_time & update_time to those POST/PUT requests sent from frontend?

Reply:
sure.